sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java

```diff
@@ -301,6 +332,66 @@ private UUID uuidFrom(QueryOuterClass.Value value) {
     }
   }
 
+  protected static final class TimestampCodec extends FromProtoValueCodec {
+    @Override
+    public Object fromProtoValue(QueryOuterClass.Value value) {
+      return Instant.ofEpochMilli(value.getInt()).toString();
+    }
+
+    @Override
+    public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
+      return jsonNodeFactory.textNode(Instant.ofEpochMilli(value.getInt()).toString());
+    }
+  }
+
+  protected static final class DateCodec extends FromProtoValueCodec {
+    @Override
+    public Object fromProtoValue(QueryOuterClass.Value value) {
+      return LocalDate.ofEpochDay(value.getDate()).toString();
+    }
+
+    @Override
+    public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
+      return jsonNodeFactory.textNode(LocalDate.ofEpochDay(value.getDate()).toString());
+    }
+  }
+
+  protected static final class TimeCodec extends FromProtoValueCodec {
+    @Override
+    public Object fromProtoValue(QueryOuterClass.Value value) {
+      return LocalTime.ofNanoOfDay(value.getTime()).toString();
+    }
+
+    @Override
+    public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
+      return jsonNodeFactory.textNode(LocalTime.ofNanoOfDay(value.getTime()).toString());
+    }
+  }
+
+  protected static final class InetCodec extends FromProtoValueCodec {
+    @Override
+    public Object fromProtoValue(QueryOuterClass.Value value) {
+      return Values.inet(value);
+    }
+
+    @Override
+    public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
+      return jsonNodeFactory.textNode(Values.inet(value).toString());
+    }
+  }
+
+  protected static final class BlobCodec extends FromProtoValueCodec {
+    @Override
+    public Object fromProtoValue(QueryOuterClass.Value value) {
+      return ByteBufferUtils.toBase64(Values.bytes(value));
```

Comment:
Hmmh. Maybe add a note that we actually have 2 choices here:

1. `byte[]` actual decoded blob contents
2. `String`: base64-encoded textual version

I think both work for intended purpose (serialization from basic Java values) but maybe worth pointing out there is a choice being made.

Reply:
I'm seeing this in the v1 code:
```
    // This covers [...] blobs (which Jackson already
    // converts to base64 natively), and time types (which are handled by registering JavaTimeModule
    // with the object mapper -- see Server.java).
    return cqlValue;
```
Why not do the same here? Just return a `ByteBuffer` and Jackson will handle it:
```
return Values.byteBuffer(value);
```